cfme/tests/infrastructure/test_publish_vm_to_template.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+
+from cfme import test_requirements
+from cfme.infrastructure.provider import InfraProvider
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.markers.env_markers.provider import ONE_PER_TYPE
+from cfme.utils.generators import random_vm_name
+from cfme.utils.log import logger
+from cfme.utils.virtual_machines import deploy_template
+from wrapanapi import VmState
+
+pytestmark = [
+    test_requirements.ownership,
+    pytest.mark.tier(2),
+    pytest.mark.provider([InfraProvider], scope='module')
+]
+
+
+@pytest.fixture(scope="module")
+def vm_crud(provider):
+    collection = provider.appliance.provider_based_collection(provider)
+    vm_name = random_vm_name(context='pblsh')
```

Comment:
Optional - 
vm_name = random_vm_name(context='publish_vm')

Reply:
context longer than 5 characters is truncated, so it's probably better to leave it like it is.